CHANGELOG/CHANGELOG-0.1.md

```diff
@@ -27,5 +27,8 @@ This release is the first release of the project. It includes the following new
 	* Arktos integration with Mizar.
 
 * Scalability
-	* Partitioned and scalable controller managers with active-active support.
-	* API Server Partitioning (in progress).
+	* Implement a controller framework that supports multiple controller instances running in active-active mode.
```

Comment:
Typically there is no ending period for list items.

Reply:
It depends. If the list item is a phrase we don't add periods (see the list in Release Summary section). But if the list item is a sentence we usually add a period.  
Of course different people may have different preferences. Personally I like this style, and it's also a recommended style:  https://www.businesswritingblog.com/business_writing/2012/01/punctuating-bullet-points-.html

Reply:
I saw many of those listed items are phrase only, not a whole sentence. The post have many comments that have different opinions.

Reply:
I will merge the PR first since this doesn't block the release and may involve some subjective views from different people. Later on if we can find an authoritative and commonly followed rule I can change that.